onc/+util/checkVersion.m

```diff
@@ -0,0 +1,33 @@
+function isLatestVersion = checkVersion()
```

Comment:
The "isLatestVersion" return from this function is not used. Perhaps this function should just not return anything?

Reply:
Will update that!